vars/publishGradleCheckTestResults.groovy

```diff
@@ -27,14 +27,15 @@ void call(Map args = [:]) {
     def buildDuration = currentBuild.duration
     def buildResult = currentBuild.result
     def buildStartTime = currentBuild.startTimeInMillis
-    def prString = isNullOrEmpty(args.prNumber.toString()) ? "post_merge_action" : "${args.prNumber}"
+    def prString = args.prNumber.toString()
```

Comment:
Is there no default if prString is null?

Reply:
From here https://github.com/opensearch-project/opensearch-build/pull/4705/files its ensured the `prString` is not null @peterzhuamazon.

Reply:
Before a null `prString` is assume to be `post_merge_action`, but it could be because a `parameterizedCron` or by a manual user trigger.

Reply:
Got it.